src/main/java/in/zapr/druid/druidry/aggregator/CardinalityAggregator.java

```diff
@@ -33,12 +33,22 @@
     private static final String CARDINALITY_AGGREGATOR_TYPE = "cardinality";
     private List<String> fields;
     private Boolean byRow;
+    private Boolean round;
 
-    @Builder
+    @Deprecated
```

Comment:
There's no need for deprecation. Adding optional field to builder will not break existing things. You can add `round` to same builder.

Reply:
I've just removed this constructor and the builder can be based on the new one